File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucUngDung.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucUngDung } from 'src/app/shared/DanhMucUngDung.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucUngDungService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucUngDung";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucUngDungService.cs
namespace Service.Implement
{
    public class DanhMucUngDungService : BaseService<DanhMucUngDung, IDanhMucUngDungRepository>
    , IDanhMucUngDungService
    {
    private readonly IDanhMucUngDungRepository _DanhMucUngDungRepository;
    public DanhMucUngDungService(IDanhMucUngDungRepository DanhMucUngDungRepository) : base(DanhMucUngDungRepository)
    {
    _DanhMucUngDungRepository = DanhMucUngDungRepository;
    }
    }
    }

